doc/whats_new/v0.24.rst

```diff
@@ -68,6 +68,9 @@ Changelog
   average of multiple RMSE values was incorrectly calculated as the root of the
   average of multiple MSE values.
   :pr:`17309` by :user:`Swier Heeres <swierh>`
+- |Enhancement| Add `sample_weight` parameter to
+  :class:`metrics.median_absolute_error`.
+  :pr:`17225` by :user:`Lucy Liu <lucyleeow>`.
```

Comment:
Space above this and use up more of the 80 characters?

Reply:
Couldn't do <2 lines without going over 80 characters.